### DISCUSSION TITLE
How do I use DSHidMini?

### DISCUSSION THREAD

**Discussion by KG240202** [nefarius/DsHidMini#302] [Q&A] [answered]

**Comment by KG240202** [comment, marked as answer]
I see that all nefarius has done is reacted rudely to my problem. Why should I even bother with these dickheads? You could have helped someone but ignored. Maybe DSHidMini isn't very useful after all...

**Reply by nefarius** [reply]
Aaaawww didn't take you long to play salty little baby. We don't need people like you here, cya never again!

**Reply by nefarius** [reply]
Imagine being so entitled that you demand a response from volunteers on the Internet...
